lib/utils/constants.dart

```diff
@@ -1,11 +1,22 @@
 abstract class Constants {
```

Comment:
As a suggestion you can create different files, one relative to all constants that you have in code, another one that only have Strings or something relative to repository/api provider

Reply:
done